dashboard_pianoroll.py

```diff
@@ -94,8 +100,19 @@ def prepare_fragments(df: pd.DataFrame, piece: MidiPiece, n: int) -> list[dict]:
                 finish_note_index=int(finish_idx),
             )
             variants.append(variant)
-        fragment = dict(variants=variants)
-        fragments.append(fragment)
+
+            # Filter out fragments with long pauses
+            max_pause_duration = 2.0
+            if finish_time - start_time > max_pause_duration:
```

Comment:
I'm not convinced that this filters out fragments with long pauses - this condition seems to be true for every variant that is longer than 2s :)